sda-svc/values.yaml

```diff
@@ -0,0 +1,292 @@
+# Default values for SDA services.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+global:
```

Comment:
The global var needs to be documented. I didn't really understand what `external` vs. `internal` actually means for example.